lib/intercom/lib/typed_json_deserializer.rb

```diff
@@ -31,6 +31,7 @@ def list_object_type?(object_type)
         end
 
         def deserialize_collection(collection_json)
+          return [] if collection_json == nil
```

Comment:
lgtm, wondering if this is hiding an error (ie is nil a reasonable value to end up with)